package.json

```diff
@@ -11,19 +11,19 @@
   },
   "homepage": "https://github.com/ckeditor/ckeditor5-dev#readme",
   "engines": {
-    "node": ">=20.0.0"
+    "node": ">=22.0.0"
   },
   "type": "module",
   "devDependencies": {
+    "@ckeditor/ckeditor5-dev-bump-year": "^49.0.1",
     "@ckeditor/ckeditor5-dev-ci": "^49.0.1",
     "@ckeditor/ckeditor5-dev-dependency-checker": "^49.0.1",
     "@ckeditor/ckeditor5-dev-release-tools": "^49.0.1",
-    "@ckeditor/ckeditor5-dev-bump-year": "^49.0.1",
     "@inquirer/prompts": "^7.4.1",
     "@listr2/prompt-adapter-inquirer": "^2.0.21",
     "@octokit/rest": "^21.0.0",
-    "eslint": "^8.21.0",
-    "eslint-config-ckeditor5": "^9.0.0",
+    "eslint": "^9.26.0",
+    "eslint-config-ckeditor5": "^10.0.0-alpha.0",
```

Comment:
Missing imports:
- `eslint-plugin-ckeditor5-rules`,
- `eslint-plugin-mocha`,
- `globals`.

Also, there is a mismatch in file masks between the `lint` script and the `lint-staged` config.